src/main/java/com/github/hanyaeger/api/engine/entities/tilemap/EntityConfiguration.java

```diff
@@ -4,36 +4,29 @@
 
 /**
  * A wrapper class that allows entities to be configured using a generic configuration object.
- *  * This configuration object can be passed to the entity's constructor for custom configuration.
+ * This configuration object can be passed to the entity's constructor for custom configuration.
+ *
  * @param <C> the type of the configuration object
  */
 class EntityConfiguration<C> {
```

Comment:
What do you think of using Lombok for data classes like these?

Reply:
Lombok nicely reduces the boileplate, but I'm actually waiting for Records to be included as a non-preview feature in one of the comming Java releases. You can read about it here: https://openjdk.java.net/jeps/359.

Its currently still in preview, but as soon as it leaves that stage, we can use it for all the data classes.